admin_app/src/app/content/edit/page.tsx

```diff
@@ -364,6 +413,15 @@ const ContentBox = ({
           return (
             <li key={option.tag_id} {...rest}>
               {option.tag_name}
+              <IconButton
+                onClick={(e) => {
+                  e.stopPropagation();
+                  openDeleteConfirmModal(option);
+                  //handleDeleteTag(option.tag_id);
+                }}
+              >
+                <Delete fontSize="small" color="secondary" />
+              </IconButton>
             </li>
```

Comment:
Nice. Can we have the delete icon be right-aligned? Something like what space-between does on flex boxes. So that the text is on the left and the delete icon is always on the right side of the list item. Pretty important for the aesthetics of it.

Also delete stray comment